src/test/groovy/com/github/vkorobkov/jfixtures/loader/RowTest.groovy

```diff
@@ -174,4 +174,60 @@ class RowTest extends Specification {
         def exception = thrown(IllegalArgumentException)
         exception.message == "Parameter <keyValuePairs> is expected to have odd length since it represents key/value pairs"
     }
+
+    def "#column returns a row"() {
+        given:
+        def row = new Row("vlad", columns)
+
+        when:
+        def result = row.column("age", 30)
+
+        then:
+        with(result.columns){
+            size() == 3
```

Comment:
Wondering if checking the size `size() == 3` is redundant when we compare text representations of Maps? `toMapString() == toMapString()`

Reply:
Actually this `size() == 3` check does not work at all, since for Spock this condition seems like:
```groovy
assert with(result.columns) {
   ...
   only the last line return result to check
}
```

If we need to have for than one assert in `with` statement, we need to define it manually:
```groovy
with(result.columns){
  assert size() == 3
  assert something_else 
  result // is auto asserted
}
```